haystack/components/evaluators/context_relevance.py

```diff
@@ -55,8 +43,8 @@ class ContextRelevanceEvaluator(LLMEvaluator):
 
     An LLM breaks up the context into multiple statements and checks whether each statement
     is relevant for answering a question.
-    The final score for the context relevance is a number from 0.0 to 1.0. It represents the proportion of
-    statements that can be inferred from the provided contexts.
+    The final score for the context relevance is either binary score of 1 or 0, where 1 indicates that the context
```

Comment:
This statement is confusing/doesn't say what the code is doing, which is calculating the mean/percentage over the entire batch. It should be updated to clarify that there are both individual scores and an aggregate.